exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillAggFuncHolder.java

```diff
@@ -93,13 +93,13 @@ public boolean isAggregating() {
         //Loop through all workspace vectors, to get the minimum of size of all workspace vectors.
         JVar sizeVar = setupBlock.decl(g.getModel().INT, "vectorSize", JExpr.lit(Integer.MAX_VALUE));
         JClass mathClass = g.getModel().ref(Math.class);
-        for (int id = 0; id < getWorkspaceVars().length; id ++) {
+        for (int id = 0; id < getWorkspaceVars().length; id++) {
           if (!getWorkspaceVars()[id].isInject()) {
             setupBlock.assign(sizeVar,mathClass.staticInvoke("min").arg(sizeVar).arg(g.getWorkspaceVectors().get(getWorkspaceVars()[id]).invoke("getValueCapacity")));
           }
         }
 
-        for(int i =0 ; i < getWorkspaceVars().length; i++) {
+        for(int i =0; i < getWorkspaceVars().length; i++) {
```

Comment:
Could you please do formatting with spaces in the lines, where you made changes? 
It may make easier adding new checkstyle rules in the future.

Reply:
@vvysotskyi I don't understand, could you give an example? I formatted the spaces on these lines because they violated the NoWhitespaceBefore rule.

Reply:
I meant also to format whole the line, for this particular case, a space after `for` should be added:
`for(int` -> `for (int`

Reply:
@vvysotskyi Updated the PR. I tried fixing any other style issues i saw with the lines I modified. Please see the latest commit.

Reply:
Thanks for fixing that, but what about my comment connected with checkstyle version? (https://github.com/apache/drill/pull/1415#discussion_r207829569)